src/pagination/mod.rs

```diff
@@ -136,23 +135,45 @@ fn interpret_permalink(
     page_num: usize,
     index: Option<&liquid::value::Value>,
 ) -> Result<String> {
-    Ok(if page_num == 1 {
-        if let Some(index) = index {
-            let include: &str = config.include.into();
-            format!("{}/{}", include, index_to_string(&index))
-        } else {
-            doc.url_path.clone()
-        }
+    let mut attributes = document::permalink_attributes(&doc.front, &doc.file_path);
+    let permalink = permalink::explode_permalink(&config.front_permalink, &attributes)?;
+    let permalink_path = std::path::Path::new(&permalink);
+    let mut pagination_root = permalink_path.extension().map_or_else(
+        || permalink.clone(),
+        |os_str| {
+            permalink
+                .trim_end_matches(&format!(".{}", os_str.to_string_lossy()))
+                .to_string()
+        },
+    );
+    // avoid "//"
+    if pagination_root.len() == 1 && pagination_root.starts_with('/') {
+        pagination_root.remove(0);
```

Comment:
If `explode` strips the leading `/`, why is this needed?

Reply:
As said above, not needed anymore :D